src/app/shell/admin-tools/data/admins/admins.component.ts

```diff
@@ -0,0 +1,94 @@
+import { MinistryAdmin } from './../../../../shared/models/ministryAdmin.model';
+import { AdminState } from 'src/app/shared/store/admin.state';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { MatTabChangeEvent } from '@angular/material/tabs';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Select, Store } from '@ngxs/store';
+import { combineLatest, Observable, Subject } from 'rxjs';
+import { filter, takeUntil} from 'rxjs/operators';
+import { AdminRole } from 'src/app/shared/enum/admins';
+import { AdminRoleUkr, AdminRoleUkrReverse } from 'src/app/shared/enum/enumUA/admins';
+import { NavBarName } from 'src/app/shared/enum/navigation-bar';
+import { NoResultsTitle } from 'src/app/shared/enum/no-results';
+import { Role } from 'src/app/shared/enum/role';
+import { PopNavPath, PushNavPath } from 'src/app/shared/store/navigation.actions';
+import { RegistrationState } from 'src/app/shared/store/registration.state';
+import { UserState } from 'src/app/shared/store/user.state';
+import { FormControl } from '@angular/forms';
+@Component({
+  selector: 'app-admins',
+  templateUrl: './admins.component.html',
+  styleUrls: ['./admins.component.scss']
+})
+export class AdminsComponent implements OnInit, OnDestroy {
+  readonly noAdmins = NoResultsTitle.noAdmins;
+  readonly adminRole = AdminRole;
+  readonly adminRoleUkr = AdminRoleUkr;
+  readonly Role = Role;
+  
+  @Select(RegistrationState.role)
+  role$: Observable<Role>;
+  role: string;
+  @Select(RegistrationState.subrole)
+  subRole$: Observable<Role>;
+  subRole: Role;
```

Comment:
I think it is redundant, I don't see where you are using it

Reply:
Done